### DISCUSSION TITLE
Smaller Docker image

### DISCUSSION THREAD

**Discussion by xavier-calland** [lukevella/rallly#1349] [Ideas]
The current Rally Docker image is over 1.4GB.
It would be nice to have a smaller image.
Is it possible to change the `runner` image from `node:20` to `node:20-slim`? This would bring the image down to 500MB.

https://github.com/lukevella/rallly/blob/3064a72979ce4ed82dbb3a9f8f939c5d086f8851/apps/web/Dockerfile#L29

**Comment by lukevella** [comment]
I tried a while ago and ran into an issue. I think it had to do with some requirement from [prisma](https://www.prisma.io/) which is the ORM we use. It's worth another shot and I would welcome a PR for this if someone can get it working with a slimmer image.

**Reply by xavier-calland** [reply]
PR : #1350
